src/main/java/org/scoverage/plugin/SCoveragePreCompileMojo.java

```diff
@@ -286,10 +284,10 @@ else if ( "2.13".equals( resolvedScalaVersion ) || resolvedScalaVersion.startsWi
 
         try
         {
-            Artifact pluginArtifact = getScalaScoveragePluginArtifact( resolvedScalaVersion, scalaBinaryVersion );
+            List<Artifact> pluginArtifacts = getScalaScoveragePluginArtifacts( resolvedScalaVersion, scalaBinaryVersion );
             Artifact runtimeArtifact = getScalaScoverageRuntimeArtifact( scalaBinaryVersion );
 
-            if ( pluginArtifact == null )
+            if ( pluginArtifacts == null )
```

Comment:
This null check looks redundant - I see no code path which dumps a null into this variable. You may want to check for an empty List though.

Reply:
I've removed this check now.